lib/api/controllers/collectionController.js

```diff
@@ -153,18 +166,18 @@ class CollectionController {
       this.kuzzle.validation.specification[request.input.resource.index] === undefined ||
       this.kuzzle.validation.specification[request.input.resource.index][request.input.resource.collection] === undefined
     ) {
-      return Promise.resolve({});
+      return Bluebird.resolve({});
     }
 
     return this.internalEngine.delete('validations', `${request.input.resource.index}#${request.input.resource.collection}`)
       .then(() => {
         this.kuzzle.pluginsManager.trigger('log:info', `Validation specification for ${request.input.resource.index}#${request.input.resource.collection} has been deleted.`);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 174 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)